ocdata/core/slab.py

```diff
@@ -71,28 +71,31 @@ def __init__(
 
     @classmethod
     def from_bulk_get_random_slab(
-        cls, bulk=None, max_miller=2, tile_and_tag=True, min_ab=8.0
+        cls, bulk=None, max_miller=2, tile_and_tag=True, min_ab=8.0, save_path=None
     ):
-        assert bulk is not None
-
-        slabs = compute_slabs(
-            bulk.atoms,
-            max_miller=max_miller,
+        all_slabs = Slab.from_bulk_get_all_slabs(
+            bulk, max_miller, tile_and_tag, min_ab, save_path
         )
```

Comment:
This would tile and tag all slabs, but only randomly pick one of them in L79, which seems quite inefficient. What's the rationale behind this?

Reply:
I missed that, will fix. I was only trying to reduce duplicate code between get random slabs and get all slabs.